### PR TITLE
Added an example using actix-web, async_graphql and diesel

### LINKED ISSUE #247: Create more examples
**Description**
It would be great to add more well-commented examples to show how to use this crate.

**Basic example**
See #245

**Other**
A new release of rust-doc should come out in some months that automatically scrap the content of our `examples` directory to show how to use each function, so adding more examples would also improve the documentation significantly.

Since the feature is still in beta right now it's hard to show you what it'll do but you can see an example [here](https://docs.rs/clap/latest/clap/trait.Parser.html#scraped-examples)

### PR DESCRIPTION
**Pull request by korir248**
# Pull Request

## Related issue
Fixes #247

## What does this PR do?
- Adds an example which depivts using `actix-web` and `async_graphql` to query records in a `postgres` database 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by korir248** [comment]
@curquiza can you take a look

File: examples/web_app_graphql/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/target
+/Cargo.lock
+
+.env
```
Comment (curquiza) [inline examples/web_app_graphql/.gitignore:4]:
```suggestion
.env
```

**Comment by irevoire** [comment]
Hey, thanks, codewise the example is impressive. That’s awesome; I’m not familiar myself with `diesel` or graph ql, so having such an example in the repository is awesome!

But when I tried to `cargo run --release` the example, I got the following error:
```
= note: ld: library not found for -lpq
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any idea why that would be the format?

**Review by irevoire — CHANGES_REQUESTED** [review]
Oh, and also, could you rename your `Readme.md` into `README.md`.

**Comment by korir248** [comment]
> Hey, thanks, codewise the example is impressive. That’s awesome; I’m not familiar myself with `diesel` or graph ql, so having such an example in the repository is awesome!
> 
> But when I tried to `cargo run --release` the example, I got the following error:
> 
> ```
> = note: ld: library not found for -lpq
>           clang: error: linker command failed with exit code 1 (use -v to see invocation)
> ```
> 
> Any idea why that would be the format?

Yes. `libpq-dev` is required when working with Postgresql. `sudo apt install libpq-dev` should fix that error

**Review by irevoire — APPROVED** [review]
Ah yes perfect I see you updated the README as well thanks a lot!

bors merge

**Pull request merged**